otsdaq-mu2e/FEInterfaces/DTCFrontEndInterfaceImpl.cc

```diff
@@ -270,29 +271,29 @@ void DTCFrontEndInterface::registerFEMacros(void)
 					// "Currently, the responses are simulated data (a counter)."
 	);
 
-	registerFEMacroFunction(
-		"Pattern Test",
-			static_cast<FEVInterface::frontEndMacroFunction_t>(
-					&DTCFrontEndInterface::PatternTest),                  // feMacroFunction
-					std::vector<std::string>{
-						"Data are SubEvents (Default: true)",
-						"Number of [Sub]Events (Default: 1)", 
-						"Starting Event Window Tag (Default: 0)",
-						"Match Event Tags (Default: false)", 
-						"Display Payload at GUI (Default: true)", 
-					// "eventDuration (Default := 400)", 
-						// "doNotReadBack (bool)", 
-						"Save Binary Data to File (Default: false)"
-						// "Software Generated Data Requests (bool)",
-						// "Do Not Send Heartbeats (bool)"
-						}, 
-					std::vector<std::string>{"response"},
-					1, // requiredUserPermissions
-					"*",
-					"Read a specified number of events from the Data DMA channel-0, and attempt to validate data."
-					// "Send a request for a number of events and waits for the respective responses. "
-					// "Currently, the responses are simulated data (a counter)."
-	);
+	// registerFEMacroFunction(
```

Comment:
#if 0 or delete

Reply:
changed to if(0) so that compiler continuously checks even though user does not see unregistered FE Macros.